integration-tests/tests/auth_test.go

```diff
@@ -0,0 +1,60 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+// +build !excludeintegration
+
+/*
+ * Copyright (C) 2015 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package tests
+
+import (
+	"github.com/ubuntu-core/snappy/integration-tests/testutils/cli"
+	"github.com/ubuntu-core/snappy/integration-tests/testutils/common"
+	"github.com/ubuntu-core/snappy/osutil"
+	"github.com/ubuntu-core/snappy/testutil"
+
+	"gopkg.in/check.v1"
+)
+
+var _ = check.Suite(&authSuite{})
+
+type authSuite struct {
+	common.SnappySuite
+}
+
+func (s *authSuite) SetUpTests(c *check.C) {
+	s.SnappySuite.SetUpTest(c)
+
+	if !osutil.FileExists("/usr/bin/curl") {
+		c.Skip("this test needs curl to work")
+	}
+}
+
+func (s *authSuite) TestRegressionAuthBypass(c *check.C) {
+	// FIXME: port to http.chipaca once that is back in action
+	cmd := []string{"curl",
```

Comment:
Good point, curl will probably never be available in ubuntu-core. However once http.chipaca is fixed it would be nice to convert the test to that so that it can run on both snappy and classic.